README.md

```diff
@@ -22,34 +22,33 @@
 
 - [📝 License](#license)
 
-
 # 📖 Testing Project <a name="about-project"></a>
 
 Testing Microverse! project is a repository consisting of the following files:
+
```

Comment:
- [optional] it's better to remove the live demo link section now to keep your project README file clean. You can add it later on when your project is deployed.
- [optional] please the future features section should include features or extra ones you will add to your application in future or time to come, not a question asked. Please look into that.